setup/asset_distribute.ts

```diff
@@ -0,0 +1,282 @@
+import {BigNumber} from 'ethers';
+import fs from 'fs-extra';
+import hre from 'hardhat';
+import {DeployFunction} from 'hardhat-deploy/types';
+import inquirer from 'inquirer';
+
+const beep = (num: number) => {
+  for (let i = 0; i < num; i++) {
+    console.log('---------------------------------------------------------');
+  }
+};
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+//require('node-beep');
```

Comment:
node-beep removed in my test branch. Do you still need it or can this comment be deleted?

Reply:
I deleted it too here from package.json. it is commented out right now in the script but we could delelte too here